scripts/configure.venv/venvactivate.pth

```diff
@@ -0,0 +1 @@
+import venvactivate
```

Comment:
what are `.pth` files used for? Was trying to figure out what that does, first searches said `pytorch`?

Reply:
They are processed by the `site` module (https://docs.python.org/3/library/site.html) which is automatically loaded by Python at startup.

If you look at the `pigweed-venv` it seems to be used by quite a few packages. The standard approach seems to be to just throw the code directly into the .pth file concatenated onto a single line, but that feels rather messy.